atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -69,8 +69,17 @@ def run(self):
         self.log.debug('getting next release from build info: %s', build_info)
         next_release = self.xmlrpc.getNextRelease(build_info)
 
+        # Always set preferred release label - other will be set if old-style 
+        # label is present
+        preferred_release_label = get_preferred_label_key(dockerfile_labels, 
+                                                         'release')
+        old_style_label = get_all_label_keys('com.redhat.component')[1]
```

Comment:
Ah ok. I finally understand now. Not sure if it's worth the effort, but we can probably put this logic in a helper method in utils. It might make it easier to read the code.